wdl/Module04b.wdl

```diff
@@ -19,23 +19,84 @@ workflow Module04b{
     Array[File] famfiles
     Array[File] RD_depth_sepcutoffs
     Int n_per_split
-    String sv_base_mini_docker
     Int n_RdTest_bins
     Array[String] batches
     Array[File] samples_lists
-    File regeno_sample_ids_lookup
-    File regeno_sample_counts_lookup 
-    File regeno_raw_combined_depth 
+    String cohort             # Cohort name or project prefix for all cohort-level outputs
+    File contig_list
     Array[File] regeno_coverage_medians
     Float regeno_max_allele_freq = 0.01 
     Int regeno_allele_count_threshold = 3 
+
+    RuntimeAttr? runtime_attr_cluster_merged_depth_beds
+    RuntimeAttr? runtime_attr_regeno_raw_combined_depth
+    RuntimeAttr? runtime_attr_regeno_merged_depth
+    RuntimeAttr? runtime_attr_sample_lookup
+    RuntimeAttr? runtime_attr_concat_samplecountlookup
+    RuntimeAttr? runtime_attr_concat_sampleidlookup
+
     RuntimeAttr? runtime_attr_vcf2bed
     RuntimeAttr? runtime_attr_merge_list
     RuntimeAttr? runtime_attr_get_count_cohort_samplelist
     RuntimeAttr? runtime_attr_get_regeno
     RuntimeAttr? runtime_attr_get_median_subset
     RuntimeAttr? runtime_attr_median_intersect
   }
+
+  call ClusterMergedDepthBeds {
+    input:
+      cohort_depth_vcf = cohort_depth_vcf,
+      cohort = cohort,
+      prefix = cohort + ".all_batches.depth",
```

Comment:
Let's get rid of this input - it doesn't actually change output file names.

Reply:
Good call, it was a bit lazy to leave it in when it's no longer relevant. Fixed and re-testing to be careful